sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1635,6 +1635,54 @@ static bool setEnvVar(const char *name, const char *value) {
   return true;
 }
 
+static class ZeUSMImportExtension {
+  // Pointers to functions that import/release host memory into USM
+  ze_result_t (*zexDriverImportExternalPointer)(ze_driver_handle_t hDriver,
+                                                void *, size_t);
+  ze_result_t (*zexDriverReleaseImportedPointer)(ze_driver_handle_t, void *);
+
+public:
+  // Whether user has requested Import/Release, and platform supports it.
+  bool USMHostPtrImportEnabled;
```

Comment:
```suggestion
  bool Enabled;
```
The uses will be through ZeUSMImport.Enabled